### PR TITLE
Add date RFC 3339

### PR DESCRIPTION
**Pull request by a-b**
RFC 3339 widely used in logging and automation

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/date.md [hunk 1]
```diff
@@ -21,3 +21,9 @@
 - Convert a specific date to the Unix timestamp format:
 
 `date -d "{{2018-09-01 00:00}}" +%s --utc`
+
+- Display RFC-3339, great for logging and timestamps YYYY-MM-DD hh:mm:ss TZ
+
+`date ---rfc-3339=s`
```
Comment (sbrl) [inline pages/common/date.md]: This example doesn't work for me on Ubuntu 19.10. What system are you using?
```
$ date ---rfc-3339=s
date: unrecognised option '---rfc-3339=s'
Try 'date --help' for more information.
```

File: pages/common/date.md [hunk 2]
```diff
@@ -21,3 +21,9 @@
 - Convert a specific date to the Unix timestamp format:
 
 `date -d "{{2018-09-01 00:00}}" +%s --utc`
+
+- Display RFC-3339, great for logging and timestamps YYYY-MM-DD hh:mm:ss TZ
```
Comment (zdroid) [inline pages/common/date.md]:
```suggestion
- Display RFC-3339 logging with timestamps:
```
Much simpler this way

File: pages/common/date.md (hunk 1 above)
Comment (zdroid) [reply pages/common/date.md]: It should be with `--`, not `---`. I edited the file to correct those pesky errors.

File: pages/common/date.md (hunk 2 above)
Comment (mebeim) [reply pages/common/date.md]: Hrm, @zdroid this does not convey the same meaning. "Display RFC-3339 logging" does not make much sense, the `date` command has nothing to do with logging *per se*, and neither does the RFC-3339, it's just a date format. Other than that, keeping the date format makes sense IMHO, that way you don't have to try the command to know the format. I would suggest the following:

```suggestion
- Display the current date using the RFC-3339 format (`YYYY-MM-DD hh:mm:ss TZ`):
```

Which is also the same way other examples in the page are worded.

Comment (zdroid) [reply pages/common/date.md]: Yes, you're right. I tried to rephrase the original, but I realized it has nothing to do with logging after running the command.

File: pages/common/date.md (hunk 1 above)
Comment (sbrl) [reply pages/common/date.md]: Ah, I see. Thanks @zdroid :-)

**Comment by zdroid** [comment]
I've edited this myself, since there was no resonse.

**Comment by a-b** [comment]
Thanks @mebeim!

**Comment by sbrl** [comment]
oops, I forgot to edit the commit message

**Pull request merged**